src/org/jitsi/impl/neomedia/rtp/sendsidebandwidthestimation/SendSideBandwidthEstimation.java

```diff
@@ -717,34 +724,49 @@ public void update(long nowMs)
         {
             synchronized (SendSideBandwidthEstimation.this)
             {
-                update(nowMs, null);
+                long time_since_packet_report_ms
+                    = nowMs - last_packet_report_ms_;
+
+                boolean currentStateHasTimedOut
+                    = time_since_packet_report_ms < 1.2 * kFeedbackIntervalMs;
```

Comment:
Perhaps extract 1.2 as a constant? It has to match the other reference to 1.2 I presume